{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/cli.py

```diff
@@ -6,136 +6,159 @@
 render model specs with the correct location parameters plugged in.
 
 It will look for the model spec template in "model_spec.in" in the directory
-"../{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/model_specification".
-Specify multiple locations in a file called "locations.txt" in the same directory.
-You can create a single location using the "-s" option. Using the single
-location option will override an existing locations file.
+``src/{{cookiecutter.package_name}}/model_specifications``.
+Specify multiple locations in a file called "locations.txt" in the same
+directory. You can create a single location using the "-s" option. Using the
+single location option will override an existing locations file.
 
 The application will look for the model spec and locations files based
 on the python environment that is active and these files don't need
 to be specified if the default names and location are used.
 """
+from collections import namedtuple
+from pathlib import Path
 import re
+from typing import List, Optional, Generator
+
 import click
-from pathlib import Path
 from jinja2 import Template
-from collections import namedtuple
 from loguru import logger
-from typing import List, Callable
-from functools import wraps
-from sys import exit
 
 
-PROJ_NAME = '{{cookiecutter.package_name}}'
-Location = namedtuple('Location', ['loc_proper', 'loc_sanitized'])
+PROJECT_NAME = '{{cookiecutter.package_name}}'
+Location = namedtuple('Location', ['proper', 'sanitized'])
```

Comment:
Simple interface thing, ``location.proper`` > ``location.loc_proper``.  I still do stuff like this all the time though.